### ISSUE TITLE
Expose a way to redraw the canvas without having to trigger an OSD redraw

### ISSUE THREAD

**Issue by iangilman** [altert/OpenSeadragonCanvasOverlay#8]
For interactive graphics or animations you might want to redraw the overlay multiple times even when the underlying OSD doesn't need updating. It would be nice to be able to trigger a draw just of the overlay. You can already do this via the `_updateCanvas` method, but the underscore marks it as private. Maybe just make it public?

See https://github.com/openseadragon/openseadragon/issues/1452 for more discussion.

**Comment by sebastianbk** [comment]
You could just use:

```js
viewer.raiseEvent('update-viewport');
```

**Comment by iangilman** [comment]
Good point! Still, it would be nice to have it as an explicit part of the API, I think.
